.appveyor.yml

```diff
@@ -29,7 +29,7 @@ environment:
   PSQL_SKIPSQLCMD: true
   PSQL_TESTDBNAME: gct_dev_ci
   
-stack: go 1.17.8 # this is not actually used on Windows images
+stack: go 1.20.3
```

Comment:
Revert as it's not used as suggested, but please add a note to change the PATH at https://github.com/thrasher-corp/gocryptotrader/blob/master/.appveyor.yml#L39 instead